samples/CallingStateful/eslint.config.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  baseConfig,
+  baseGlobalIgnores,
+  globalConfig,
+  testFilesBaseConfig
+} from '../../common/config/ESLint/eslint.config';
+import { globalIgnores } from 'eslint/config';
+
+export default globalConfig([
+  baseConfig,
+  testFilesBaseConfig,
+  {
+    rules: {
+      'headers/header-format': 'off',
```

Comment:
Same here, the original config has the header format rule. 
Why disable it now?

Reply:
I see whats going on here. Ill fix this up.